unetsocket/js/test/spec/unet.spec.js

```diff
@@ -195,3 +195,36 @@ describe('A UnetSocket', function () {
 
 });
 
+describe('Unet Utils', function () {
+  function coordTester(res, val) {
+    const tolerance = 0.000100;
+    const min = res - tolerance;
+    const max = parseFloat(res) + tolerance;
+    return (val >= min && val <=max);
+  }
+
+  beforeEach(function() {
+    jasmine.addCustomEqualityTester(coordTester);
+  });
+
+  it('should be able to convert local coordinates to GPS coordinates', function () {
+    const origin=[1.34286,103.84109], x=100, y=100;
```

Comment:
Did we cross-check these numbers with the Java implementation?

Reply:
Yes tested using below,
```
async function conv(){
  let testloc = await xyToLatLng(origin,[100,100]);
    console.log('Using toGPS of ShellExecReq :'+testloc);
    testloc = await toGps(origin,100,100);
    console.log('Using toGPS of Unetjs :'+testloc);
    testloc = await latlngToXY(origin,[1.343764,103.841988]);
    console.log('Using toLocal of ShellExecReq:'+testloc);
    testloc = await toLocal(origin,1.343764,103.841988);
    console.log('Using toLocal of Unetjs:'+testloc);
}
```
Below was the result:
```
Using toGPS of ShellExecReq :1.343764364515331,103.84198856067962
xy-latllng.html:43 Using toGPS of Unetjs :1.343764364515331,103.84198856067962
xy-latllng.html:45 Using toLocal of ShellExecReq:99.93760247532074,99.95969376010984
xy-latllng.html:47 Using toLocal of Unetjs:99.93760247532074,99.95969376010984
```